package.json

```diff
@@ -24,17 +24,17 @@
     "README.md"
   ],
   "dependencies": {
-    "@typescript-eslint/utils": "^6.10.0",
+    "@typescript-eslint/utils": "^7.4.0",
     "tsutils": "^3.21.0"
```

Comment:
If you have the time, swapping this to `ts-api-utils` would be nice. `@typescript-eslint/*` packages already use `ts-api-utils` anyway so it'd just be removing an old, no-longer-maintained dependency.